src/Microsoft.Health.Dicom.SqlServer/Features/Schema/SchemaVersion.cs

```diff
@@ -11,5 +11,6 @@ namespace Microsoft.Health.Dicom.SqlServer.Features.Schema
     public enum SchemaVersion
     {
         V1 = 1,
+        V2 = 2,
```

Comment:
I know we discussed this in the standup. Are we keeping this in a Feature branch untill we have initial schema of everything? Since this is OSS now, we need to make sure users can upgrade.

Reply:
The plan is to keep this on feature branch until the min scope is completed. Then merge to main branch.
min scope include: Get/List/Add/Delete custom tag(s), support custom tag on QIDO, STOW. Without the reindex feature.

We can talk if you have concerns.